src/azure-cli/azure/cli/command_modules/containerapp/_transformers.py

```diff
@@ -18,6 +19,41 @@ def transform_containerapp_output(app):
     return result
 
 
+def clean_up_sensitive_values(response_json):
+    for container in safe_get(response_json, "properties", "template", "containers", default=[]):
+        if "env" in container:
+            for env in container["env"]:
+                del env["value"]
+
+    if "scale" in response_json["properties"]["template"]:
+        for rule in safe_get(response_json, "properties", "template", "scale", "rules", default=[]):
+            for (key, val) in rule.items():
+                if key != "name":
+                    val["metadata"] = dict((k, "") for k, v in val["metadata"].items())
+    return response_json
+
+
+def transform_sensitive_values_wrapper(args):
```

Comment:
It may not be a good idea to do transformation based on the command line `args`. Consider `--show-sensitive-values` has an alias, such as `-a`, L38 would then need to check both `--show-sensitive-values` and `-a`. 

Performing transformation in `custom.py` by checking canonicalized `show_sensitive_values` seems more reasonable.